.github/workflows/tests_and_formatting.yml

```diff
@@ -76,7 +74,5 @@ jobs:
       - name: Check if test succeeded
         if: steps.test.outcome != 'success'
         run: exit 1
-      - name: Prettify JS code
-        uses: creyD/prettier_action@v4.3
-        with:
-          prettier_options: --write --tab-width 4 /home/runner/work/fastn/fastn/fastn-js/js/**/*.js
+      - name: Check if JS code is properly formatted
+        uses: dprint/check@v2.2
```

Comment:
Do one more thing, use `config-path: .github/dprint-ci.json` and move the `dprint.json` to `.github` to not pollute our root folder.